packages/core/src/App/app.jsx

```diff
@@ -66,42 +55,10 @@ const AppWithoutTranslation = ({ root_store }) => {
         setUrlLanguage(getLanguage());
         initFormErrorMessages(FORM_ERROR_MESSAGES);
         setSharedCFDText(CFD_TEXT);
-        handleResize();
         root_store.common.setPlatform();
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    const handleResize = React.useCallback(() => {
-        if (isTouchDevice() && (isMobile() || isTablet())) {
-            const is_android_device = mobileOSDetect() === 'Android';
-            const view_width = is_android_device ? screen.availWidth : window.innerWidth;
-            const view_height = is_android_device ? screen.availHeight : window.innerHeight;
-            const el_landscape_blocker = document.getElementById('landscape_blocker');
-
-            if (
-                view_width <= view_height ||
-                root_store.modules?.cashier?.general_store?.is_user_on_p2p ||
-                window.location.pathname.endsWith(routes.cashier_p2p)
-            ) {
-                root_store.ui.onOrientationChange({ is_landscape_orientation: false });
-                el_landscape_blocker.classList.remove('landscape-blocker--visible');
-            } else {
-                root_store.ui.onOrientationChange({ is_landscape_orientation: true });
-                el_landscape_blocker.classList.add('landscape-blocker--visible');
-            }
-        }
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [root_store.ui]);
-
-    React.useEffect(() => {
-        const debouncedHandleResize = debounce(handleResize, 400);
-        window.addEventListener('resize', debouncedHandleResize);
-
-        return () => {
-            window.removeEventListener('resize', debouncedHandleResize);
-        };
-    }, [handleResize]);
-
```

Comment:
why are we removing these?

Reply:
there are a lot of issues with the way we are handling the landscape blocker for various browsers and devices. a big problem is when the soft keyboard appears and sometimes itll resize the whole viewport itself. this will cause the landscape blocker to appear even though we're still on portrait mode... Also, this function is listening to every window resize event, whereas we should just detect if the user is on landscape mode. When I removed this and checked if the user is in landscape using css, it switched faster and there are no issues across other devices or browsers